### DISCUSSION TITLE
using an old database as source for new inventree installation

### DISCUSSION THREAD

**Discussion by lobra07** [inventree/InvenTree#6689] [Q&A] [answered]
hello all 

my pc once crashed and i had to reinstall windows 
i didn't have any backup  of my inventree 
 i have the data directory of the inventree old installation 

my deployment was docker 
any idea on how to use my old data with new installation of docker 

thanks in advance

**Comment by SchrodingersGat** [comment]
You may be able to follow the [data migration guide](https://docs.inventree.org/en/stable/start/migrate/)

However if your old installation is not up and running this will be tricky.

If you have the "data" directory I would suggest the following:

1. Setup a new docker installation - https://docs.inventree.org/en/stable/start/docker_install/
2. Ensure all containers are stopped - `docker compose down`
3. *Copy* your *old* data directory and overwrite the *new* data directory (copy, don't just move)
4. Run `docker compose --rm inventree-server invoke update`
5. Fingers crossed...

Please post the output of that here. And make sure you keep an un-corrupted copy of your original data dir

**Comment by lobra07** [comment, marked as answer]
thank you for the fast response 

i did try it in different way 


first Setup a new docker installation
then containers are stopped - docker compose down 

then set env variables to the same as they were in the old setup 

INVENTREE_DB_ENGINE=postgresql
INVENTREE_DB_NAME=inventree_old_db
INVENTREE_DB_HOST=inventree-db
INVENTREE_DB_PORT=5432
INVENTREE_DB_USER=database_old_user
INVENTREE_DB_PASSWORD=database_old_user_password


then i copied the PGDB file and MEDIA file into the new installation 
then  Docker compose up -d 

and everything worked perfectly all data are back 


thanks
